data/assets/scene/solarsystem/planets/earth/lagrange_points/l2/l2sunline.asset

```diff
@@ -0,0 +1,44 @@
+local sun = asset.require("scene/solarsystem/sun/sun")
+local sunTransforms = asset.require("scene/solarsystem/sun/transforms")
+local transforms = asset.require("./transforms")
+
+
+
+local L2SunLine = {
+  Identifier = "L2SunLine",
+  Parent = sunTransforms.SolarSystemBarycenter.Identifier,
+  Renderable = {
+    Type = "RenderableNodeLine",
+    Enabled = asset.enabled,
+    StartNode = sun.Sun.Identifier,
+    EndNode = transforms.L2Position.Identifier,
+    Color = { 0.5, 0.5, 0.5 },
+    LineWidth = 2
+  },
+  Tag = { "lagrange_points_earth_l2_small" },
```

Comment:
This tag seems to be wrong. This is the node with the sun to L2 line, not the small L2 indicator. Not sure what we want instead, maybe just remove it? Or at least add the `"lagrange_points_earth"` general tag?